src/redux/sagas/actions/addVerifiedMembers.ts

```diff
@@ -1,46 +1,43 @@
-import { ClientType } from '@colony/colony-js';
-import { fork, put, takeEvery } from 'redux-saga/effects';
+import { ClientType, ColonyRole, getPermissionProofs } from '@colony/colony-js';
+import { call, fork, put, takeEvery } from 'redux-saga/effects';
 
-import { apolloClient } from '~apollo';
-import { GetColonyContributorsDocument } from '~gql';
-import { ActionTypes } from '~redux';
-import type { Action, AllActions } from '~redux';
+import type ColonyManager from '~context/ColonyManager';
+import { type Action, type AllActions, ActionTypes } from '~redux';
 import { transactionAddParams } from '~redux/actionCreators/transactions.ts';
 
 import {
   createTransaction,
   createTransactionChannels,
-  waitForTxResult,
-} from '../transactions/index.ts';
+  getTxChannel,
```

Comment:
this entire file seems like a bad rebase, please revert it to the way it's on the feature branch